src/DotNetWorker.Grpc/FunctionMetadata/DefaultFunctionMetadataProvider.cs

```diff
@@ -12,6 +12,7 @@
 
 namespace Microsoft.Azure.Functions.Worker
 {
+    [DefaultMetadataProvider]
```

Comment:
Marked this one for now. Another option could be to use a the full type name instead?